src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/HttpClientLoggingServiceCollectionExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static class HttpClientLoggingServiceCollectionExtensions
     /// All other loggers are removed - including the default one, registered via <see cref="HttpClientBuilderExtensions.AddDefaultLogger(IHttpClientBuilder)"/>.
     /// </remarks>
     /// <exception cref="ArgumentNullException">Argument <paramref name="services"/> is <see langword="null"/>.</exception>
-    public static IServiceCollection AddDefaultHttpClientLogging(this IServiceCollection services)
+    public static IServiceCollection AddExtendedHttpClientLogging(this IServiceCollection services)
```

Comment:
So, now both names are the same?
`IServiceCollection.AddExtendedHttpClientLogging` and `IHttpClientBuilder.AddExtendedHttpClientLogging`
Wouldn't it confuse users?

Reply:
IMO as long as `serviceCollection.AddExtendedHttpClientLogging()` has the same behavior as `services.ConfigureHttpClientDefaults(httpBuilder => httpBuilder.AddExtendedHttpClientLogging())` then I wouldn't expect significant confusion. Worst case if a customer was aware of both and confused by it there is still no wrong answer, whichever one they try will work.

@davidfowl I think has stronger opinions that we should avoid offering IServiceCollection extension methods when a sub-builder extension method also exists. Personally I'm neutral about this pattern, but assuming I didn't misunderstand his position, he'd probably want us not to include the IServiceCollection variant.